test/stresstest/client.py

```diff
@@ -0,0 +1,48 @@
+########################################################################
+# Copyright (c) 2020 Robert Bosch GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+########################################################################
+
+import kuksa_client
+import config
+
+class Client:
+
+    def start(self):
+        clientType = self.cfg.get("protocol")
+        if clientType == 'ws':
+            print('Starting VISS client')
+            try:
+                self.connect()
+                self.commThread.authorize(token=self.commThread.backend.tokenfile)
+            except:
+                raise Exception
+        elif clientType == 'grpc':
+            print('Starting gRPC client')
+            try:
+                self.connect()
+            except:
+                raise Exception
+        else: 
+            raise ValueError
```

Comment:
```suggestion
            raise ValueError(f"Unknown client type {clientType}")
```
Context is always better when raising exceptions. Also, it's preferable to instantiate the exception type here i.e. `raise ValueError()` instead of `raise ValueError` (without `()`).